cliquet/tests/resource/test_pagination.py

```diff
@@ -1,7 +1,8 @@
+import mock
 import random
-from base64 import b64encode, b64decode
+import six
```

Comment:
In python3 mock is, but the difference I do is more Python functionality versus Project specific functionality